src/patternfly/components/Toolbar/examples/Toolbar.md

```diff
@@ -334,7 +334,7 @@ The `.pf-m-toggle-group` controls when, and at which breakpoint, filters will be
 | Class | Applied to | Outcome |
 | -- | -- | -- |
 | `.pf-m-[show/hide][-on-[sm/md/lg/xl/2xl]]` | `.pf-v6-c-toolbar__group.pf-m-toggle-group`, `.pf-v6-c-toolbar__expandable-content` | Modifies toolbar element visibility. |
-| `.pf-m-chip-container` | `.pf-v6-c-toolbar__content-section`, `.pf-v6-c-toolbar__group` | Modifies the toolbar element for applied filters layout. |
+| `.pf-m-label-container` | `.pf-v6-c-toolbar__content-section`, `.pf-v6-c-toolbar__group` | Modifies the toolbar element for applied filters layout. |
```

Comment:
Should be label-group-container. Looks like we're also missing documentation on just `pf-m-label-group`